index.html

```diff
@@ -4,9 +4,49 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href="stylesheets/Components/landing.css"/>
+    <link rel='stylesheet' href="stylesheets/CSS/reset.css"/>
+    <link rel='stylesheet' href="stylesheets/CSS/typography.css"/>
+    <link rel='stylesheet' href="stylesheets/CSS/variables.css"/>
```

Comment:
Suggestion: Instead of importing css stylesheets of each component. Why not use another main.css file which has all the imports. Then you'll only need to import one file. 

In a scenario of a page where you need multiple components. You'll need to import a lot of css files..but if they all are imported in a single main.css then only main.css can be imported into your html.

Reply:
I did that because alerts page was aligning itself in the landing page layout and causing conflict.... i have imported a single file of style.css for all the components part. If there is a better way to do this other than what i have done.... please do tell, as i had done it exactly the same way you are telling.. and i didn't know any better way to solve the conflict that arised. 

Thanks for the feedback.❤️